ngraph/core/include/openvino/core/static_shape.hpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstdio>
+#include <numeric>
+#include <vector>
+
+#include "openvino/core/attribute_adapter.hpp"
+#include "openvino/core/axis_set.hpp"
+#include "openvino/core/core_visibility.hpp"
+#include "openvino/core/strides.hpp"
+
+namespace ov {
+/// \brief StaticShape for a tensor.
+class StaticShape : public std::vector<size_t> {
```

Comment:
can we replace to `std::vector<int64_t>` while old good `ngraph::Shape` has `std::size_t`?

Reply:
I tried to do it, but a lot of structures uses ngraph::Shape and it doesn't look backward compatible. I propose to return to the question with the `get_partial_shape()` solution